org.eclipse.scout.rt.dataobject/src/main/java/org/eclipse/scout/rt/dataobject/id/IRootId.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2010-2022 BSI Business Systems Integration AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     BSI Business Systems Integration AG - initial API and implementation
+ */
+package org.eclipse.scout.rt.dataobject.id;
+
+/**
+ * An interface to represent an arbitrary id with cardinality one. The type of the raw (wrapped) id is not further
+ * specified. See various sub-interfaces for commonly used wrapped id types.
+ * <p>
+ * Subclasses must provide the following static factory methods:
+ * <ul>
+ * <li><b>of(wrapped-type)</b>: used by {@link IdFactory} to construct new instances. The method is expected to return
+ * <code>null</code> if the given {@code value} is <code>null</code>, otherwise the wrapped value.
+ * </ul>
+ * <p>
+ *
+ * @see IUuId
+ * @see IStringId
+ * @see ILongId
+ * @see AbstractRootId
+ */
+public interface IRootId extends IId {
+
+  /**
+   * @return the raw id formatted as string.
```

Comment:
Is the implementor completely free in defining a string representation or are there any restrictions which should be stated as JavaDoc? The only purpose here should be to use as debug info instead of a more verbose toString().

Reply:
Added javadoc pointing to correct usage of unwrap method